internal/engine/upper.go

```diff
@@ -374,6 +377,14 @@ func handleLogTimestampsAction(state *store.EngineState, action hud.SetLogTimest
 	state.LogTimestamps = action.Value
 }
 
+func handleSailRoomConnectedAction(ctx context.Context, state *store.EngineState, action client.SailRoomConnectedAction) {
+	if action.Err != nil {
+		logger.Get(ctx).Infof("Error connecting Sail room: %v\n", action.Err)
```

Comment:
it always seems weird to me when reducers log, since that just pushes out another action, tho i'm not totally sure what we should be doing instead

Reply:
🤷‍♀️ gooood question.